docs/src/whatsnew/3.6.rst

```diff
@@ -13,12 +13,23 @@ This document explains the changes made to Iris for this release
    :animate: fade-in
    :open:
 
-   The highlights for this major/minor release of Iris include:
+   We're so excited about our recent support for **delayed saving of lazy data
+   to netCDF** (:pull:`5191`) that we're celebrating this important step change
+   in behavour with its very own dedicated release 🥳
 
-   * N/A
+   We're super keen for the community to leverage the benefit of this new
+   feature within Iris that we've brought this release forward several months.
+   As a result, this minor release of Iris is intentionally light in content.
+   However, there are some other goodies available for you to enjoy, such as:
 
-   And finally, get in touch with us on :issue:`GitHub<new/choose>` if you have
-   any issues or feature requests for improving Iris. Enjoy!
+   * Performing lazy arithmetic with an Iris :class:`~iris.cube.Cube` and a
+     :class:`dask.array.Array`, and
+   * Various improvements to our documentation including dark-mode friendly
```

Comment:
We've starting making preparations for dark-mode, but #5299 is still outstanding at the moment.

Reply:
It was in reference to #5278, but I'll drop that comment :+1: